hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -590,7 +588,7 @@ void setDataBlockEncoderInTest(HFileDataBlockEncoder blockEncoder) {
         }
       }
     } finally {
-      storeFileOpenerThreadPool.shutdownNow();
+
```

Comment:
If nothing to cleanup, remove the finally block?

Reply:
> If nothing to cleanup, remove the finally block?

Ok, let me update the patch a moment later